aspnetcore/signalr/background-service/samples/3.x/Clients.ConsoleTwo/ClockHubClient.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using HubServiceInterfaces;
+using Microsoft.AspNetCore.SignalR.Client;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Clients.ConsoleTwo
+{
+    public class ClockHubClient : IClock, IHostedService
+    {
+#region ClockHubClientCtor
+        private readonly ILogger<ClockHubClient> _logger;
+        private HubConnection _connection;
+
+        public ClockHubClient(ILogger<ClockHubClient> logger)
+        {
+            _logger = logger;
+            
+            _connection = new HubConnectionBuilder()
+                .WithUrl(Strings.HubUrl)
+                .Build();
+
+            _connection.On<DateTime>(Strings.Events.TimeSent, 
+                dateTime => _ = ShowTime(dateTime));
```

Comment:
`On` handlers should support async methods.
```suggestion
                dateTime => ShowTime(dateTime));
```

Reply:
Resolved in b552e67